### PR TITLE
Created models typescript for the database

### PR DESCRIPTION
**Pull request by nomaddevs1**
Added Typescript Models based on the ERD  diagram provided
TODO: We still need to add the methods to the models for full functionality.

### CODE REVIEW

File: server/models/employee.model.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export interface Employee {
+  id: number;
+  email: string;
+  full_name: string;
+  phone_number: string;
+  reliability_score: number;
```
Comment (shaham-noorani) [inline server/models/employee.model.ts:6]: lets make this optional for now

File: server/models/employee_shirt.model.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+enum size {
```
Comment (shaham-noorani) [inline server/models/employee_shirt.model.ts:1]: lets make this `Size`

File: server/models/event_volunteer.model.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+enum status {
```
Comment (shaham-noorani) [inline server/models/event_volunteer.model.ts:1]: lets make this `Status`

File: server/models/event_volunteer.model.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  REJECTED,
+  WAILISTED,
+  CANCELLED,
+}
+
+export interface EventVolunteer {
+  id: number;
+  event_id: number;
+  employee_id: number;
+  did_atend: boolean;
```
Comment (shaham-noorani) [inline server/models/event_volunteer.model.ts:13]: did_attend (missing a t)

**Review by shaham-noorani — CHANGES_REQUESTED** [review]

File: server/models/employee.model.ts (hunk 1 above)
Comment (nomaddevs1) [reply server/models/employee.model.ts]: Fixed

File: server/models/employee_shirt.model.ts (hunk 2 above)
Comment (nomaddevs1) [reply server/models/employee_shirt.model.ts]: Fixed

**Review by shaham-noorani — APPROVED** [review]

**Pull request merged**